src/components/Organisms/Todocard/index.jsx

```diff
@@ -29,6 +29,16 @@ const TodoCard = () => {
     }
     setTasklist(newTasklist);
   };
+  useEffect(() => {
+    const data = JSON.parse(localStorage.getItem("tasklist"));
+
+    if (data === null) return;
+    setTasklist(data);
```

Comment:
早期リターンよいですね 👍 
```suggestion
    const data = JSON.parse(localStorage.getItem("tasklist"));

    setTasklist(data ?? []);
```
既存実装に対する利点はあまりありませんが、こう書く手もあるなと思いました。